BraintreeModels/src/main/java/com/braintreepayments/api/SamsungPayConfiguration.java

```diff
@@ -67,8 +56,7 @@ static SamsungPayConfiguration fromJson(JSONObject json) {
      * @return {@code true} if Samsung Pay is enabled, {@code false} otherwise.
      */
     boolean isEnabled() {
-        return !"".equals(mSamsungAuthorization) &&
-                classHelper.isClassAvailable(SAMSUNG_PAY_CLASSNAME);
+        return !TextUtils.isEmpty(mSamsungAuthorization);
```

Comment:
This was causing a test to fail. We feel like the responsibility to check at runtime if Samsung is available on the classpath should go to a `SamsungClient`.